projectile.el

```diff
@@ -1342,9 +1350,16 @@ Fallback to a generic command when not in a VCS-controlled project."
 Currently that's supported just for Git (sub-projects being Git
 sub-modules there)."
   (pcase vcs
-    ('git projectile-git-submodule-command)
+    ('git (and projectile-git-submodule-command-function (funcall projectile-git-submodule-command-function)))
     (_ "")))
 
+(defun projectile-get-git-sub-projects-command ()
```

Comment:
Probably this should be marked as private and also explain why the command has to be generated at runtime.

Reply:
Does this make sense?
```emacs-lisp
(defun projectile--get-git-sub-projects-command ()
  "Get the sub-projects command for Git.
Defined as a function in order to generate the properly-quoted command at
runtime."
```